docs/BR.md

```diff
@@ -1848,15 +1848,15 @@ If the CA asserts compliance with these Baseline Requirements, all certificates
 | Signed Certificate Timestamp List | MAY             | N            | See [Section 7.1.2.11.3](#712113-signed-certificate-timestamp-list) |
 | Any other extension               | NOT RECOMMENDED | -            | See [Section 7.1.2.11.5](#712115-other-extensions) |
 
-##### 7.1.2.1.3 Authority Key Identifier
+##### 7.1.2.1.3 Root CA Authority Key Identifier
 
 | __Field__                   | __Description__ |
 | ---                         | ------- |
 | `keyIdentifier`             | MUST be present. MUST be identical to the `subjectKeyIdentifier` field. |
 | `authorityCertIssuer`       | MUST NOT be present |
 | `authorityCertSerialNumber` | MUST NOT be present |
 
-##### 7.1.2.1.4 Basic Constraints
+##### 7.1.2.1.4 Root CA Basic Constraints
```

Comment:
Same as above comment (both regarding change justification and my opinion of the change).